### DISCUSSION TITLE
Why mongoose pins Mongodb to 4.5.0 instead of allowing upgrades?

### DISCUSSION THREAD

**Discussion by huksley** [Automattic/mongoose#11801] [General]
i.e. package.json contains

```js
"mongodb": "4.5.0",
```

As a result, installing the latest MongoDB driver creates mongodb@4.5.0 as a private package inside mongoose/node_modules
which results in type problems with user code and Mongoose?

Is there anything wrong with MongoDB 4.6.0 driver?

**Comment by Uzlopak** [comment]
There is nothing wrong with MongoDB 4.6.0 driver

Please read:
https://github.com/Automattic/mongoose/discussions/11494#discussioncomment-2678491

**Reply by huksley** [reply]
Hmm, when I install mongodb@4.6.0 i will have 2 packages of MongoDB in node_modules. Could package.json say `"mongodb": "^4.5.0",`

I am not sure how I can override this via npm/package.json

**Reply by Uzlopak** [reply]
We pin the versions in mongoose to avoid issues, when mongodb gets some kind of regression/breaking change. 

Why do you even need mongodb as a dependency in your project, when you already use mongoose?

Anyway:
https://stackoverflow.com/a/48524488

**Reply by huksley** [reply]
I use Mongoose for my code and I also use MongoDB as an adapter in Next Auth

Thanks for link, I probably need to upgrade to npm-cli v.8
